tools/reviewer/web/src/index.html

```diff
@@ -7,6 +7,7 @@
   <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
   <meta name="viewport" content="width=device-width, initial-scale=1">
   <link rel="icon" type="image/x-icon" href="favicon.ico">
+  <link rel="stylesheet" href="/assets/highlight-github-gist.css">
```

Comment:
You can include it in the angular-cli.json file. 
Just include the path to the css file directly in the styles array